scripts/ci/manage-operator.sh

```diff
@@ -171,19 +171,23 @@ install_operator() {
         INDEX_IMAGE=${INDEX_IMAGE_LOC}
         CHANNEL=staging
     fi
-    CATALOGSOURCE_NAME=source-${OPERATOR_NAME}-${GIT_COMMIT_ID}
-    SUBSCRIPTION_NAME=subscription-${OPERATOR_NAME}-${GIT_COMMIT_ID}
+    CATALOGSOURCE_NAME=${OPERATOR_RESOURCE_NAME}-${GIT_COMMIT_ID}
+    SUBSCRIPTION_NAME=${OPERATOR_RESOURCE_NAME}-${GIT_COMMIT_ID}
 
     # if the operator was already installed in the cluster, then delete all OLM related resources
-    for SUB in $(oc get Subscription -n ${NAMESPACE} -o name | grep  "subscription-${OPERATOR_NAME}"); do
+    for OG in $(oc get OperatorGroup -n ${NAMESPACE} -o name | grep  "${OPERATOR_RESOURCE_NAME}"); do
+        oc delete ${OG} -n ${NAMESPACE}
```

Comment:
doesn't the oc delete need the type of the resource when deleting by name?
`oc delete OperatorGroup ${OG} -n ${NAMESPACE}`

Reply:
TBH I doesn't seem to be required, it seems to be working for me. I'll give it another try tomorrow.

I see also the other ( existing ) commands are using the same syntax 🤷‍♂️

Reply:
Actually I've just verified why it is working - it is because the `grep` in the previous line returns also the api name:
```
operatorgroup.operators.coreos.com/host-operator
```
thus we should be fine.

Reply:
Thanks for rechecking!